airbyte-integrations/connectors/source-pagerduty/metadata.yaml

```diff
@@ -4,7 +4,7 @@ data:
       - api.pagerduty.com
   remoteRegistries:
     pypi:
-      enabled: true
+      enabled: false
       packageName: airbyte-source-pagerduty
   registryOverrides:
     oss:
```

Comment:
Wait, why the hell is it not enabled on any registries? Can you please enable it, too?